src/editor/components/settings/token-list-item.tsx

```diff
@@ -0,0 +1,21 @@
+import { ListItem } from '../list/list-item'
+import { Button } from '../../components/form/button'
+
+export interface TokenListItemProps {
+  name: string;
+  id: string;
+  deleteToken( name: string ): void
+}
+
+export const TokenListItem = ( { name, id, deleteToken }: TokenListItemProps ): JSX.Element => (
+  <ListItem>
+    <>
+      <div>
+        <div className="font-semibold dark:text-gray-100">{name}</div>
```

Comment:
I'd like something like this:
![image](https://user-images.githubusercontent.com/7260168/168171691-27229758-0c5f-44d5-8a47-59302fed3d4c.png)

Use a monospace font, green color and a lock icon before the name. You can use the Heroicon package that is already used in the editor for that.